servers/physics_3d/space_3d_sw.cpp

```diff
@@ -34,6 +34,8 @@
 #include "core/config/project_settings.h"
 #include "physics_server_3d_sw.h"
 
+#define TEST_MOTION_MIN_CONTACT_DEPTH_FACTOR 0.05
```

Comment:
Yeah, I've decided to use the same factor on both to make it simple. So it's a bit higher in 3D and it doesn't seem to cause any issue. Actually it might help filter out wrong contact points in some cases.